.travis.yml

```diff
@@ -19,6 +19,12 @@ addons:
   apt:
     update: true
 
+before_install:
+  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+  - sudo apt-get update -qq
+  - sudo apt-get install --yes -qq gcc-6
```

Comment:
no need to have gcc-6, installed gcc version is sufficient.

Reply:
Done